src/utils/urlHelpers.ts

```diff
@@ -0,0 +1,50 @@
+import { TODOIST_WEB_URI } from '../consts/endpoints'
+
+/**
+ * Generate the URL for a given task.
+ *
+ * @param taskId The ID of the task.
+ * @param content The content of the task.
+ * @returns The URL string for the task view.
+ */
+export function getTaskUrl(taskId: string, content?: string): string {
+    const slug = content ? slugify(content) : undefined
+    const path = slug ? `${slug}-${taskId}` : taskId
+    return `${TODOIST_WEB_URI}/task/${path}`
```

Comment:
With the main `TodoistApi` class and some of the other helper functions, we can pass in an optional base url so that the SDK can work with other environments, I wonder whether similar should be added to these helper functions too.

Reply:
I accounted for this when I introduced `TODOIST_WEB_URI` and hardcoded it in the URL helpers. The main reason is that `baseUrl` doesn't always point to the frontend domain. It's typically the API entrypoint. The hardcoded [`BASE_URI`](https://github.com/Doist/todoist-api-typescript/blob/c0280f0b1a2d5d15dbe40aeaf2ffe446280c8a9c/src/consts/endpoints.ts#L1) is `https://api.todoist.com`, and the web app isn’t accessible through that domain (e.g. `https://api.todoist.com/app/` doesn’t exist).

That’s why I’m leaning towards keeping the generated URL domains non-customizable. Let me know if you have a different take on this.

Reply:
Might be ok then. I guess it would be more of an issue for generating the oauth url.